recipes/regtools/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "0.3.0" %}
+package:
+  name: regtools
+  version: {{ version }}
+
+source:
+  fn: "{{ version }}.tar.gz"
+  md5: c2af56b0350b6f2b88e5ce52ab5b8721
+  url: https://github.com/griffithlab/regtools/archive/{{ version }}.tar.gz
+
+requirements:
+  build:
+  - gcc  # [not osx]
+  - llvm # [osx]
+  - cmake
+  run:
+  - libgcc # [not osx]
+about:
+  home: https://github.com/griffithlab/regtools
+  license: MIT
+  summary: "Tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context."
+
+test:
+  commands:
+  - which regtools
```

Comment:
i will make the default output into a test, though i have a question regarding that. 

the tool's 0.3.0 release erroneously says '0.2.0' in its --help output. this is fixed in a non-release commit (several commits after release), but is confusing to anyone who would run --help. i assume i should stick with the release version, but should I make a note of the inconsistency somewhere?